### DISCUSSION TITLE
Two Factor Authentication has been added to the Epic Stack

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#155] [Announcements]
[Watch the tip](https://www.epicweb.dev/tips/two-factor-auth-is-included-in-the-epic-stack): https://www.epicweb.dev/tips/two-factor-auth-is-included-in-the-epic-stack

<a href="https://www.epicweb.dev/tips/two-factor-auth-is-included-in-the-epic-stack"><img width="2560" alt="image" src="https://github.com/epicweb-dev/epic-stack/assets/1500684/49cbca56-6aa2-445d-9f7b-fd82c4aad822"></a>

Two factor auth is built on top of the same TOTP support added earlier this week: https://github.com/epicweb-dev/epic-stack/discussions/148

Enjoy!

**Comment by andrecasal** [comment]
Amazing Kent, thank you so much for this! 🎉
